tests/testthat/test-cube-subset.R

```diff
@@ -129,11 +185,7 @@ test_that("translateCubeIndex", {
         translateCubeIndex(cat_x_mr_x_mr, alist(1:2, 1, 2), drop = FALSE),
         alist(1:2, 1, TRUE, 2, TRUE)
     )
-    # MR selection entries are set to index 1 when the indicator is dropped
-    expect_identical(
-        translateCubeIndex(cat_x_mr_x_mr, alist(1:2, 1, 2), drop = TRUE),
-        alist(1:2, 1, "mr_select_drop", 2, "mr_select_drop")
-    )
```

Comment:
Was this test deletion intentional?

Reply:
It was but on reflection I think it might still be useful so I put it back.